oneflow/core/graph/graph.h

```diff
@@ -148,6 +149,10 @@ class Graph {
                          const std::string& file_path) const;
   void ToDotWithAutoFilePath() const;
 
+ protected:
+  void ProtectedDestoryNodes();
```

Comment:
```suggestion
  void ProtectedDestoryNodes();
  void DestoryNodes();
```
不需要额外在函数名中加 Protected，其他地方的代码没有这个习惯